easydigitaldownloads/easy-digital-downloads
# Update row selector for repeatable row cloning

Fixes #8195

Proposed Changes:
1. Updates the row selector in the EDD JavaScript for wider backwards compatibility with extensions.

To test:
1. Check out release/3.0 and the master branch of Software Licensing (3.6.12).
2. Edit a download and scroll down to the Upgrade Paths metabox.
3. Click the "Add New" button (it will not work).
4. Switch to this branch and reload the screen (may require a hard refresh).
5. Clicking the "Add New" button should now work.
